src/models/playbook/PlaybookModel.js

```diff
@@ -50,10 +59,16 @@ export class PlaybookModel {
      */
     printJsContent(indentSize = 1)
     {
-        let indent = TextIndentService.indent(indentSize);
+        let indent = TextService.indent(indentSize);
+
 
         let content = 'const path = require("path");\n\n';
-            content += 'playbook("' + this.name + '")\n';
+
+        this.stepModels.forEach((stepModel) => {
+            content += "const " + stepModel.name + " = require('./" + stepModel.path + "\');\n";
```

Comment:
I am very happy with this approach. 

A backlog task *could* be switched between builder pattern and imperative pattern based on size of commits 

I think this is unnecessary